server/eb/README.md

```diff
@@ -0,0 +1,65 @@
+# AWS Elastic Beanstalk 
+
+This directory contains script to aid in creating and deploying cellxgene on
+an AWS Elastic Beanstalk instance.
```

Comment:
It might be good to add a quick summary of what this will provide, once deployed.  For example:

> This will result in a variant of cellxgene, running on AWS EC2 instances, serving data from S3.   All datasets must be in the new CXG (tiledb) format - see the converter script `cxgtool.py` located ??? - and located in a single S3 prefix, which is accessible to the instance.   In the current incarnation, no access control or authentication support is available (outside of anything you configure yourself), so this is most appropriate for public datasets.

> This is early development work, and will change significantly in the near future.   We would love feedback on it, but please assume it will change.

Reply:
Looks good, I'll add that.